job_creator.php

```diff
@@ -132,10 +132,24 @@ public function getInstallerVersion(
 
     public function createJob(int $phpIndex, array $opts): array
     {
+        $isInstallerModule = in_array($this->repoName, INSTALLER_MODULES);
+        $isPreferLoweset = strpos($opts['composer_args'] ?? '', '--prefer-lowest') !== false;
+        $installerVersion = $this->installerVersion;
+        $cmsMajor = BranchLogic::getCmsMajor($this->repoData, $this->branch, $this->getComposerJsonContent());
+        if ($cmsMajor >= 5 && !$isInstallerModule && $isPreferLoweset && $installerVersion) {
```

Comment:
```suggestion
        if (!$isInstallerModule && $isPreferLoweset && $installerVersion) {
```
There's no reason to exclude CMS 4 here - those should all have compatible dependencies already (and if they don't, that's a problem)

Reply:
The reason it's excluded is because in the corresponding gha-ci PR each major will have the following minor releases:
`x.0, x.1, x.2, x.3, x.4`

The way this solution works it needs to start at `x.0`. CMS 4 goes all the was up to 4.13. I don't really want it to waste lots of time incrementing all the way up to x.13 if something just isn't going to install.

Given that CMS 4 isn't going to suffer from '--prefer-lowest' style issues at this point, I thing it's best to just  exclude it